### ISSUE TITLE
HUE lux sensor integration

### ISSUE THREAD

**Issue by luscaGIT** [karloygard/hassio-addons#22]
Hi

Can this integration be used to take the LUX value of a HUE outdoor sensor and trigger an action in alrady existing xComfort system to active ZIP Screens ?
5000lx+ value down, less then 3000lx zips do up.. ?

**Comment by karloygard** [comment]
Should be possible. I have no shutters myself, so I have no experience with them, but give it a shot and report back. If there are issues, I'm pretty sure we can work them out.

**Comment by luscaGIT** [comment]
Ok, I see in the install it the itegration ask for an authkey.  I only have the USB-CKOZ-00/13. 
Do you have an idea where I could get the key in question?

**Comment by karloygard** [comment]
You will need the CKOZ-00/14, should be readily available online.

**Comment by luscaGIT** [comment]
Ah ok, thats annoying since I already have the 13.. 
I might aswell just get a Aotec Z-wave stick 7 and plug it into the rasp with HA, add it with Z-Wave JS to MQTT in HA. 
Then add it in sensio xSetup...

In my head that would work flawless, but in reality. I dont know.
Couple weeks wait time for a  Aotec 700series stick

**Comment by karloygard** [comment]
Yes, annoying "feature" of the xComfort system that you need both the 13 and 14 sticks to be fully up and running.
